### ISSUE TITLE
Missing "NotInSub" / "InSub" in CacheKey generator types

### ISSUE THREAD

**Issue by smartssa** [GeneaLabs/laravel-model-caching#149]
**Describe the bug**
Using a NotIn with a function call to create a subquery causes a crash. Adding "NotInSub" and "InSub" to the array in CacheKey.php#100 makes the crash go away, but it doesn't appear to invalidate the cache properly when the subquery data changes.

**Eloquent Query**

```php
        return Challenge::whereNotIn('id', function ($query) use ($userId) {
                $query->select('challenges_id')->from('users_has_challenges')->where('users_id', '=', $userId);
            })
            ->orderBy('created_when', 'DESC')
            ->limit($limit)
            ->get();

```

**Stack Trace**
```
ErrorException(code: 0): Undefined index: operator at /Volumes/Storage/Git/triviosity-2.0-web/vendor/genealabs/laravel-model-caching/src/CacheKey.php:102
```

**Environment**
 - PHP: 7.2.5
 - OS: macOS 10.11.6
 - Laravel: 5.6.28
 - Model Caching: 0.2.63

**Comment by mikebronner** [comment]
@smartssa Thanks for reporting this! I will look into this as soon as I can -- hopefully by this weekend.

**Comment by mikebronner** [comment]
Fixed in version 0.2.64. @smartssa Please let me know how it goes.

**Comment by smartssa** [comment]
Thanks for the quick fix!
